icbc-adapter/src/Pssg.Dmf.IcbcAdapter/Controllers/IcbcController.cs

```diff
@@ -147,8 +148,7 @@ public ActionResult CreateCandidates([FromBody] List<NewCandidate> newCandidates
                             CommentText =  $"This case was opened because a DMER was issued to this driver by ICBC on {dmerIssuranceDate}",
                             CommentTypeCode = "C",
                             UserId = "System",
-                             
-                            // dfp_origin 100000001
+                            Assignee = assignee
```

Comment:
Can you add ?? string.empty here to protect against Null.

Also, please add similar code to the Legacy Adapter for situations where a Comment is sent by GRPC